tests/02_generic_input/08_field_init_check.yaml

```diff
@@ -55,6 +55,7 @@ problem: !Coupling_Sequential
         - velocity_p0
```

Comment:
Test default for the stream in 03_generic_output dir. Add comment in description.